03/app.js

```diff
@@ -1,58 +1,67 @@
 const books = {
-    '978-83-7278-000-3': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Philosopher\'s Stone',
-            pl: 'Harry Potter i Kamień Filozoficzny',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
-    },
-    '83-7278-007-2': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Chamber of Secrets',
-            pl: 'Harry Potter i Komnata Tajemnic ',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
-    },
-}
+	'978-83-7278-000-3': {
+		author: 'J.K. Rowling',
+		title: {
+			en: "Harry Potter and the Philosopher's Stone",
+			pl: 'Harry Potter i Kamień Filozoficzny',
+		},
+		translator: {
+			en: null,
+			pl: 'Andrzej Polkowski',
+		},
+	},
+	'83-7278-007-2': {
+		author: 'J.K. Rowling',
+		title: {
+			en: 'Harry Potter and the Chamber of Secrets',
+			pl: 'Harry Potter i Komnata Tajemnic ',
+		},
+		translator: {
+			en: null,
+			pl: 'Andrzej Polkowski',
+		},
+	},
+};
 
-books.getAuthor = function(isbn) {
-    // w tym przypadku this === books
-    
-    if(typeof this[isbn] === 'undefined') {
-        // taki ISBN nie istnieje w moim spisie
-        return null;
-    }
+books.getAuthor = function (isbn) {
+	// w tym przypadku this === books
+	if (typeof this[isbn] === 'undefined') {
+		// taki ISBN nie istnieje w moim spisie
+		return null;
+	}
 
-    const author = this[isbn]['author'];
-    if(author) {
-        // zwracam informacje o autorze
-        return author;
-    }
+	const author = this[isbn]['author'];
+	if (author) {
+		// zwracam informacje o autorze
+		return author;
+	}
 
-    // brak informacji o autorze
-    return false;
-}
+	// brak informacji o autorze
+	return false;
+};
 
-books.getTitle = function(isbn, lang) {
+books.getTitle = function (isbn, lang) {
+	const language = lang;
+	const isTitleAvailable = this[isbn]['title'][language];
```

Comment:
Najpierw trzeba sprawdzić czy ` this[isbn]` istnieje - inaczej już na tym etapie będzie błąd :)